providers/shared/flows/components/flow.ftl

```diff
@@ -570,27 +693,208 @@
                         [/#list]
                     [/#list]
 
+                    [#-- Determine any provided location information permitting profiles to contribute content --]
+                    [#local locations =
+                        getCompositeObject(
+                            [
+                                {
+                                    "Names" : ["deployment:Locations"],
```

Comment:
I think there is actually a configuration section where the deployment  global variable exists. Would it be possible to put this in there to keep the deployment configuration in one spot?

Reply:
It also aligns with the model that some component/subcomponents can't be split and some can. The deployment details should only show up on components that have control over their deployment.

Reply:
There is a special "deployment" provider in the default RG where they all are. Given where this lookup is in the code, I didn't want to be looking up all the deployment info. 

Even if you can't deploy an occurrence, it can still have locations so perhaps I need a different prefix. Open to suggestions ...

Reply:
I think it might be better to move it to a different prefix. The deployment options shouldn't be included by default. I think some of them are at the moment

Reply:
OK - I'll go with `placement:` as that is the predominant use of locations. We could also use them for things like code repos - not sure if that sort of thing is better done as links on the component or as an addition location. Advantage of locations is they do check that the endpoint is the correct type whereas in the solution it is just whether the link is formatted correctly.